bucket/goldendict.json

```diff
@@ -17,11 +25,16 @@
         "portable",
         "content"
     ],
-    "checkver": {
-        "url": "https://sourceforge.net/p/goldendict/activity/feed",
-        "regex": "GoldenDict-([\\w.-]+)_\\(QT_486\\)\\.7z"
-    },
+    "checkver": "github"
+    ,
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/goldendict/early%20access%20builds/GoldenDict-$version_(QT_486).7z"
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/goldendict/goldendict/releases/download/$version/GoldenDict-${version}_.QT_$matchBuild.64bit.7z"
+            },
+            "32bit": {
+                "url": "https://github.com/goldendict/goldendict/releases/download/$version/GoldenDict-${version}_.QT_$matchBuild.7z"
```

Comment:
```suggestion
                "url": "https://github.com/goldendict/goldendict/releases/download/$version/GoldenDict-$version_.QT_$matchBuild.7z"
```

Reply:
···
> $version = 'balabala'
> "$version_"

> "${version}_"
balabala_
```

Reply:
Because "/" is not a legal variable character, while "_" is.
```
> $version = 'balabala'
> "/$version/"
/balabala/
```
I have checked and I have successfully installed GoldenDict with the local manifest.
In fact, there are a lot of mainfests in this bucket 'Extras' that use "checkver": "github", "to check the version, and I don't know why you use the following code to check the version.
```
"checkver": {
    "url": "https://api.github.com/repositories/1864049/releases",
    "jsonpath": "$..browser_download_url",
    "regex": "GoldenDict-([\\d.]+)_\\.QT_(?<build>[\\d]+)\\.64bit\\.7z"
    },
```